### PR TITLE
Event Hub : update Title in readme and added location parameter to GeoDR template

### PR DESCRIPTION
**Pull request by v-Ajnava**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* updated Title in readme files 
* added location parameter to GeoDR template (geo Dr requires namesapces in to different regions)

### CODE REVIEW

File: 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.parameters.json [hunk 1]
```diff
@@ -13,6 +13,12 @@
     },
 	"aliasName": {
 	  "value": "GEN-UNIQUE"	
-	}
+	},
+	"locationSecondaryNamepsace": {
```
Comment (bmoore-msft) [inline 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.parameters.json]: remove these and use the defaultValue in the template

File: 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json [hunk 2]
```diff
@@ -40,7 +40,21 @@
       ],
       "defaultValue": 1,
       "metadata": {
-        "description": "MessagingUnits for premium namespace"
+        "description": "MessagingUnits for namespace"
+      }
+    },
+	"locationPrimaryNamepsace": {
+      "type": "string",
+      "defaultValue": "North Central US",
```
Comment (bmoore-msft) [inline 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json]: use northcentralus (ie remove spaces)

File: 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json [hunk 3]
```diff
@@ -40,7 +40,21 @@
...
+      "metadata": {
+        "description": "Location of Primary namespace"
+      }
+    },
+	"locationSecondaryNamepsace": {
+      "type": "string",
+      "defaultValue": "South Central US",
```
Comment (bmoore-msft) [inline 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json]: use southcentralus

Comment (v-Ajnava) [reply 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json]: Resolved

File: 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json (hunk 2 above)
Comment (v-Ajnava) [reply 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.json]: Resolved

File: 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.parameters.json (hunk 1 above)
Comment (v-Ajnava) [reply 201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.parameters.json]: Resolved

**Pull request merged**